packages/arb-token-bridge-ui/src/components/common/DepositCountdown.tsx

```diff
@@ -12,14 +13,28 @@ function getMinutesRemainingText(minutesRemaining: number): string {
 }
 
 function getEstimatedDepositDurationInMinutes(
-  parentChainId: number | undefined
+  tx: MergedTransaction,
+  considerOnlyFirstTransactionForTeleport?: boolean
```

Comment:
NIT: Maybe we could do something like `firstTxOnly` flag. Then we don't have teleport specific flag as it fits standard transfers too (even tho they only have one transaction)

Reply:
Updated